app/org/sagebionetworks/bridge/models/studies/PasswordPolicy.java

```diff
@@ -0,0 +1,85 @@
+package org.sagebionetworks.bridge.models.studies;
+
+import java.util.Objects;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * The policies that govern password creation. New studies will be created with the default password 
+ * policy, which requires 8 characters, one numeric, one symbol and one upper- and lower-case 
+ * character. However, this policy can be adjusted for legacy studies which were created with much 
+ * more lax constraints.
+ */
+public final class PasswordPolicy {
+    
+    public static final int FIXED_MAX_LENGTH = 100;
```

Comment:
We discussed on IRC... I believe there's always some value in Stormpath as you can't set it to zero. 100 should be enough.